src/cpp/r/R/Tools.R

```diff
@@ -1120,7 +1173,7 @@ environment(.rs.Env[[".rs.addFunction"]]) <- .rs.Env
       base <- baseList[[name]]
       overlay <- overlayList[[name]]
       if (is.list(base) && is.list(overlay) && recursive)
-        mergedList[[name]] <- merge_lists(base, overlay)
```

Comment:
This is one of those "I can't believe this ever worked" kind of bugs ... but you can reproduce with:

```
> .rs.mergeLists(
+    list(a = list(b = list(c = 42))),
+    list(a = list(b = list(c = 43)))
+ )
Error in merge_lists(base, overlay) : 
  could not find function "merge_lists"
```

I guess we didn't use this to merge recursive lists very often!